src/App.tsx

```diff
@@ -1,10 +1,19 @@
 import { ConfigProvider } from '@4chain-ag/react-configuration';
-import { AuthProvider, SpvWalletProvider, ThemeProvider, useAuth } from '@/contexts';
+import { AuthProvider, SpvWalletProvider, ThemeProvider, useAuth, useSpvWalletClient } from '@/contexts';
 import { createRouter, RouterProvider } from '@tanstack/react-router';
 import { routeTree } from '@/routeTree.gen.ts';
+import { TooltipProvider } from '@/components/ui/tooltip.tsx';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+
+const queryClient = new QueryClient();
 
 // Create a new router instance
-const router = createRouter({ routeTree, context: { auth: undefined! } });
+const router = createRouter({
+  routeTree,
+  context: { auth: undefined!, queryClient, spvWallet: undefined! },
```

Comment:
Why do we need the contexts defined in router?

Reply:
To have access to these contexts in Route object

Reply:
Could you show me where it is used?

Reply:
It will be used in other routes object. Using `loader` in `Route` object